nginx/run-benchmark.sh

```diff
@@ -0,0 +1,121 @@
+# This script has no hashbang as it's intended to run on systems with ash or
+# bash, including busybox.
+set -eu
+
+. /usr/local/bin/output.sh
+
+usage() {
+  cat <<EOF
+Usage: run-benchmark.sh [-h] [-v] [-y benchmark_type] [-t threads]
+                        $(output_options_short)
+
+Run the sysbench benchmark
```

Comment:
So, this directory didn't have a run-benchmark before, because it was only running nginx, with no benchmark. The wrk benchmark was in the wrk directory. If you're going to copy an existing file, you should use the wrk one, not the sysbench one.

Reply:
What are you suggesting here? Should just `s/sysbench/wrk/g`? Is there something to do with flags? Could you please elaborate what I should change?

Reply:
Well, I don't really know how this file was created, but I'm assuming Kautilya copied it from the sysbench benchmark and modified it. I'm saying the one from the wrk one should have been copied and modified, as that one already included the wrk parameters and configs...

On the other hand, I don't know why we aren't using the benchmark already setup in the wrk folder instead of this one. From what I gathered, that was some sort of misunderstanding of the fact that we wanted to benchmark "nginx".

Reply:
> From what I gathered, that was some sort of misunderstanding of the fact that we wanted to benchmark "nginx".

Yes, I guess this is the case. I thought we wanted to benchmark nginx so added it